### ISSUE TITLE
Some questions about training results

### ISSUE THREAD

**Issue by PyAllegro** [KaoruMuta/ISIC2016_challenge#1]
Your open source content is very good.  but I met some questions when I try to run the program according to your instructions，I find that my results are generally different from any experimental results in your result directory. I run the train_ISIC2016.py program with trainModel、 batchSize = 32、trainingEpochs=20、 plt、 cutout and focal settings.Some of the results are as follows：
![accuracy](https://user-images.githubusercontent.com/38966664/58608967-78a2e900-82d8-11e9-97b3-96cdbe9e7384.png)
![confusion_matrix](https://user-images.githubusercontent.com/38966664/58608968-793b7f80-82d8-11e9-8b3c-f10cab282b63.png)
![loss](https://user-images.githubusercontent.com/38966664/58608969-793b7f80-82d8-11e9-994a-6b07ca12a88d.png)
![roc_curve](https://user-images.githubusercontent.com/38966664/58608971-793b7f80-82d8-11e9-9d21-b74b806a6549.png)
[accuracy.txt](https://github.com/KaoruMuta/ISIC2016_challenge/files/3235452/accuracy.txt)
I don't know what the problem is, so can you give me some advice? Thank you very much！

**Comment by KaoruMuta** [comment]
Thanks for watching my content and sorry for the late reply.

I made a mistake on the statement in README. I achieved the best result by using only focal loss and setting batchsize as 16 probably. 
Could you try to implement again?

Best

**Comment by PyAllegro** [comment]
First of all, thank you for taking the time to answer my questions.

Secondly, I have another question: when you use Focalloss loss function for training, what do you choose for the pre-training model, densenet161 or resnet101? I'm using resnet101, but there's a problem with actual training, which is that the model doesn't seem to converge steadily. I'm not sure why, as shown in the data recorded in the txt file below.
Looking forward to your reply

Regards


[train_loss.txt](https://github.com/KaoruMuta/ISIC2016_challenge/files/3283800/train_loss.txt)
[train_acc.txt](https://github.com/KaoruMuta/ISIC2016_challenge/files/3283801/train_acc.txt)

**Comment by KaoruMuta** [comment]
I used SE-ResNeXt-101 as model.
